source/postprocess/visualization/seismic_melt.cc

```diff
@@ -0,0 +1,372 @@
+                                 "Units: $GPa$.");
+
+              prm.declare_entry ("Liquid bulk modulus", "40",
+                                 Patterns::Double (),
+                                 "The bulk modulus of the liquid. "
+                                 "Units: $GPa$.");
+
+              prm.declare_entry ("Shear modulus", "64",
+                                 Patterns::Double (),
+                                 "The shear modulus of the solid material "
+                                 "assuming no porosity. "
+                                 "Units: $GPa$.");
+
+              prm.declare_entry ("Poisson ratio", "0.25",
+                                 Patterns::Double (),
+                                 "The Poisson ratio of the solid material. "
+                                 "Units: non-dimensional.");
+
+              prm.declare_entry ("p1", "-8065.00",
+                                 Patterns::Double (),
+                                 "The first coefficient, associated with "
+                                 "porosity to the fifth power, in the fifth "
+                                 "order polynomial contiguity equation "
+                                 "of Wimert and Hier-Majumder (2012).");
+
+              prm.declare_entry ("p2", "6149.00",
+                                 Patterns::Double (),
+                                 "The second coefficient, associated with "
+                                 "porosity to the fourth power, in the fifth "
+                                 "order polynomial contiguity equation "
+                                 "of Wimert and Hier-Majumder (2012).");
+
+              prm.declare_entry ("p3", "-1778.00",
+                                 Patterns::Double (),
+                                 "The third coefficient, associated with "
+                                 "porosity to the third power, in the fifth "
+                                 "order polynomial contiguity equation "
+                                 "of Wimert and Hier-Majumder (2012).");
+
+              prm.declare_entry ("p4", "249.00",
+                                 Patterns::Double (),
+                                 "The fourth coefficient, associated with "
+                                 "porosity to the second power, in the fifth "
+                                 "order polynomial contiguity equation "
+                                 "of Wimert and Hier-Majumder (2012).");
+
+              prm.declare_entry ("p5", "-19.77",
+                                 Patterns::Double (),
+                                 "The fifth coefficient, associated with "
+                                 "porosity to the first power, in the fifth "
+                                 "order polynomial contiguity equation "
+                                 "of Wimert and Hier-Majumder (2012).");
+
+              prm.declare_entry ("p6", "1.00",
```

Comment:
This should always be one, correct?
If the porosity approaches 0, you want the contiguity to approach 1. I guess anything else wouldn't make any sense physically, would it? If that's the case, just don't make it a variable at all, but just replace it by 1.0 in line 144.